config/wallet-config.schema.json

```
@@ -0,0 +1,68 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "type": "object",
+  "description": "Dictionary describing the application versions for which the messages should be displayed. Use `global` for all versions.",
+  "patternProperties": {
```

The upper level doesn't need to be pattern properties. It can just be: 

```suggestion
  "properties": {
```

This is only needed for `messages`, as a key could be any version number.